public/index.html

```diff
@@ -34,6 +34,12 @@
     <title>
       ReactPlay - One app to learn, create, and share ReactJS projects.
     </title>
+    <script
+      async
+      defer
+      data-website-id="e9be9998-e02c-48ee-b0e4-8456ca5c57a1"
```

Comment:
What is this id related to? Is it any sensitive info to come from the env file?

Reply:
@atapas , This is the unique id created by umami.

Reply:
> @atapas , This is the unique id created by umami and yes we should take it from .env

Reply:
ok thanks, let's fix then.